src/auth/feedback-secret.guard.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  Injectable,
+  CanActivate,
+  ExecutionContext,
+  UnauthorizedException,
+} from '@nestjs/common';
+import { Env } from 'src/env';
+
+@Injectable()
+export class FeedbackSecretGuard implements CanActivate {
+  constructor() {}
```

Comment:
PrimathonAICodeReviewer

The constructor in the `FeedbackSecretGuard` class is empty and can be safely removed.

```diff
-   constructor() {}
```



<!-- This is an auto-generated comment by OSS PrimathonAICodeReviewer -->